src/layout/Navbar.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {NavLink} from "react-router-dom";
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome'
+import {faBars, faDog} from "@fortawesome/free-solid-svg-icons";
+import {MOBILE_SCREEN_SIZE} from "../Constants";
+
+
+const Navbar = () => {
+    const [show, setShow] = React.useState(false);
+
+    let w = document.documentElement.clientWidth || window.innerWidth;
```

Comment:
@allispeck, can we use Tailwindcss's css breakpoints instead? Might run into some issues later on that could impact how you would construct your components :)
https://tailwindcss.com/docs/responsive-design

Reply:
Awesome!